test/routes/index-test.js

```diff
@@ -1,21 +1,25 @@
 const { expect } = require('chai');
 const got = require('got');
 
-const { server } = require('../../src/index');
+const App = require('../../src/app');
 
 describe('Index page', () => {
-    before(async () => {
-        await server.start();
+    const apps = [3001, 3002, 3003].map(port => new App({ port }));
+
+    before(() => {
+        apps.forEach(app => app.run());
     });
 
-    after(async () => {
-        await server.stop();
+    after(() => {
+        apps.forEach(app => app.close());
     });
 
-    it('should show "Hello Node.js" message', async () => {
-        const response = await got(`http://${server.host}:${server.port}`);
+    it('all apps should show "Hello Node.js" message', async () => {
```

Comment:
Well, it's hard to imagine a situation when 3 apps can pass a tests and just a single app can't. But I think there is a-kind-of rule that says that we shouldn't remove tests when it is not necessary.